src/main/java/se/sundsvall/businessrules/rule/impl/parkingpermit/criteria/ExpiringParkingPermitCriteria.java

```diff
@@ -0,0 +1,96 @@
+package se.sundsvall.businessrules.rule.impl.parkingpermit.criteria;
+
+import static generated.se.sundsvall.citizenassets.Status.ACTIVE;
+import static generated.se.sundsvall.citizenassets.Status.EXPIRED;
+import static java.time.LocalDate.MAX;
+import static java.time.LocalDate.now;
+import static java.util.Collections.emptyList;
+import static java.util.Optional.ofNullable;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.springframework.util.CollectionUtils.isEmpty;
+import static se.sundsvall.businessrules.integration.citizenassets.CitizenAssetsClient.PARTY_ID_PARAMETER;
+import static se.sundsvall.businessrules.integration.citizenassets.CitizenAssetsClient.TYPE_PARAMETER;
+import static se.sundsvall.businessrules.rule.impl.parkingpermit.enums.ParkingPermitFactKeyEnum.STAKEHOLDERS_APPLICANT_PERSON_ID;
+import static se.sundsvall.businessrules.service.mapper.RuleEngineMapper.toFactMap;
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import generated.se.sundsvall.citizenassets.Asset;
+import se.sundsvall.businessrules.api.model.Fact;
+import se.sundsvall.businessrules.integration.citizenassets.CitizenAssetsClient;
+import se.sundsvall.businessrules.rule.Criteria;
+import se.sundsvall.businessrules.rule.CriteriaResult;
+
+/**
+ * Criteria for expired or almost expired parking permits.
+ *
+ * Inputs:
+ * stakeholders.applicant.personid
+ */
+@Component
+public class ExpiringParkingPermitCriteria implements Criteria {
+
+	private static final String HAS_EXPIRING_PARKING_PERMITS = "den sökande har parkeringstillstånd som strax upphör eller redan har upphört";
+	private static final String HAS_NO_EXPIRING_PARKING_PERMITS = "den sökande har inga parkeringstillstånd som är på väg att upphöra eller redan har upphört";
+	static final int EXPIRATION_PERIOD_IN_MONTHS = 2;
```

Comment:
Change to private and create explicit test verification variable to catch unintentional changes to this value (something that won't be caught if this attribute is used in tests)?